src/http/endpoints/query_entity.rs

```diff
@@ -0,0 +1,31 @@
+use crate::ayb_db::db_interfaces::AybDb;
+use crate::ayb_db::models::InstantiatedEntity;
+use crate::error::AybError;
+use crate::http::permissions::can_query;
+use crate::http::structs::{EntityDatabase, EntityQueryPath, EntityQueryResponse};
+use crate::http::utils::unwrap_authenticated_entity;
+use actix_web::{get, web};
+
+#[get("/v1/entity/{entity}")]
+pub async fn query_entity(
```

Comment:
So that people don't get confused and think it's related to DB queries

```suggestion
pub async fn entity_details(
```

Reply:
You're right